pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -558,6 +567,7 @@ func (p *PriorityQueue) Update(oldPod, newPod *v1.Pod) error {
 					return err
 				}
 				p.unschedulablePods.delete(usPodInfo.Pod)
+				klog.V(5).InfoS("Pod moved", "pod", klog.KObj(pInfo.Pod), "event", "PodUpdated", "from", unschedulablePods, "to", activeQName)
```

Comment:
(not related to this line)

Add logging below L564? unschedulable -> backoff

Reply:
Added additional logging `unschedulePods` to `backoff`.

Reply:
> what about when the pods leave the queues?
A scheduling event has the timestamp like 

**I0817 00:54:33.456604 1495316** scheduling_queue.go:597] "Pod moved" pod="yuan/pod-143-c95674667-h6t7b" event="NodeLabelChange" from="UnschedulableQueue" to="BackoffQueue"